package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "copy-word",
-  "displayName": "Copy Word in Cursor",
-  "description": "Copy/Cut the current word when there is no selection",
-  "version": "3.6.0",
+  "displayName": "Copy/Paste Word in Cursor",
+  "description": "Cut/Copy/Paste the current word when there is no selection",
+  "version": "3.7.0",
```

Comment:
Undo these changes. I'm not sure the new `Paste` command needs to update the extension's name/description